Headers/DebugServer2/Base.h

```diff
@@ -113,4 +115,13 @@ typename std::enable_if<
                           reinterpret_cast<uintptr_t>(&array[0]));
 }
 
+namespace ds2 {
+
+// We don't use C++14 (yet).
+template <typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args &&... args) {
```

Comment:
Apparently it was "partly an oversight": https://herbsutter.com/gotw/_102/

On Mon, Jun 20, 2016 at 1:11 PM, walter erquinigo <notifications@github.com>
wrote:

> In Headers/DebugServer2/Base.h
> <https://github.com/facebook/ds2/pull/383#discussion_r67757617>:
>
> > @@ -113,4 +115,13 @@ typename std::enable_if<
> >                            reinterpret_cast<uintptr_t>(&array[0]));
> >  }
> >
> > +namespace ds2 {
> > +
> > +// We don't use C++14 (yet).
> > +template <typename T, typename... Args>
> > +std::unique_ptr<T> make_unique(Args &&... args) {
>
> i just wonder why they didn't implement this in c++11
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/383/files/0235db2bd70835cdbd35eb0a149546ec2ad25a22#r67757617>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbDUJ44KvC3x9ksexUZrnjksS5qo8ks5qNvQFgaJpZM4I4Eje>
> .
>



-- 
Stephane Sezer